designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -376,11 +374,13 @@ class EditProjectCtrl {
             });
         }
         if (this.form.teamPrune) {
-            // this.form.teamPrune = projectData.teamMembers;
             this.form.teamPrune.forEach((ent) => {
                 projectData.teamMembers.push(ent.username);
             });
         }
+        if (!projectData.teamMembers.concat(projectData.coPis, [projectData.pi]).includes(this.form.creator.username)) {
```

Comment:
Since "creator" in this component is always the current user, this makes it so that you can't ever remove yourself from a project even if you didn't create it. Is this intended?

Reply:
This was intentional. We wanted to make it so users could not accidentally remove themselves from a project, but now that I'm thinking about it, we could flash a warning or something saying "Hey you're about to remove yourself from the project" before continuing.